### DISCUSSION TITLE
Displaying Results

### DISCUSSION THREAD

**Discussion by aswinchari** [connectomicslab/connectomemapper3#70] [Q&A] [answered]
Hello,

I'm trying to use SurfIce to display some of my results from analyses. Is there a nii version of the default Lausanne 2018 atlas (eg for MNI brain or something) that I can use for this? 

My other option is to use one of the subjects in my dataset.

Thanks,
Aswin

**Comment by sebastientourbier** [comment, marked as answer]
Hi Aswin,
Unfortunately such a nii version of the Lausanne 2018 parcellations in MNI space is not provided yet.
But good news, there exists a solution. You can process a small sample BIDS dataset that includes the Colin27 template (http://www.bic.mni.mcgill.ca/ServicesAtlases/Colin27) which is in MNI space such that you can get Lausanne 2018 parcellations in MNI space.

**Comment by aswinchari** [comment]
Thanks Sebastien,

I pretty much did that - I ran the paediatric average brain template through connectome mapper and got the parcellation - works very nicely!

Aswin

**Comment by sebastientourbier** [comment]
Great!!! 
You're welcome! 
So I am closing this issue.
